### DISCUSSION TITLE
2023년 04월 27일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#47] [General]
# 2023년 04월 27일

LOG 장바구니 미션 1단계 PR을 보냈다.  @ControllerAdvice  를 사용해서 에러 처리를 추가했다. 갑자기 예외 나온 이유를 프론트에서 보여주고 싶어서 js, html도 수정했다. 글렌이 만든  ErrorResponse  를 참고해서…

https://cyma-s.github.io/2023-04-27/

**Comment by Choi-JJunho** [comment]
기술부채 나중에 뺏어먹으러 가야겠다 🦟
